paraatm/tests/test_para_atm.py

```diff
@@ -105,7 +110,7 @@ def test_vcas(self):
                'sim_time': 1000}  # total simulation time
 
         sim = VCAS(cfg)
-        track = sim()['trajectory']
+        track = sim()
```

Comment:
In this commit, it looks like you inadvertently reverted some previous changes.  I think you realized this later because you fixed these in a subsequent commit.  Probably what you did is after merging the upstream changes, you copied and pasted a file with your edits, which were based off of an older version of the file.  By doing this, you essentially "undid" some of the upstream changes, such as this one.

The best way to avoid this is to always use `git status` and `git diff` prior to making commits.  You can review exactly what changes in the file you are about to commit.  When you notice that you have inadvertently introduced changes that should not be committed, there are a couple of ways to handle this.  Git provides a command `git checkout --patch` that you can use to interactively "accept and reject" individual changes within the file, but this is a little bit more advanced.  Other options are to manually change the file back or "start over" with a clean version (making sure you have a separate copy of your changes first): `git checkout -- filename` will remove all of your local changes to the file, and then you can selectively copy/paste in the actual changes that you make individually, instead of copying the entire file, which would overwrite other changes that had been made.